docs/components/table.md

```diff
@@ -257,6 +264,111 @@ export default {
 </script>
 ```
 
+## Custom Row Attributes
+
+There is a prop called `hasSideBorder` that when passed in, enables a default side border on each row, using the color `steal-300` by default.
```

Comment:
Please move this to the top props section https://github.com/Kong/kongponents/pull/231/files#r449151214

Reply:
I also don't think you need to call out the default color. Just say its customizable with a theme var